packages/cli/src/Ldap/helpers.ts

```diff
@@ -497,3 +497,8 @@ export const transformAllLdapUsersToEmailUsers = async () => {
 export const disableAllLdapUsers = async () => {
 	return Db.collections.User.update({ signInType: SignInType.LDAP }, { disabled: true });
 };
+
+export const isInstanceOwner = async (email: string): Promise<boolean> => {
+	const user = await Db.collections.User.findOne({ email });
+	return user && user?.globalRole?.name === 'owner' ? true : false;
```

Comment:
for this to work you need to change this to 
```suggestion
    const user = await Db.collections.User.findOne({ where: { email }, relation: ['globalRole']});
	return user && user?.globalRole?.name === 'owner' ? true : false;
```

Reply:
This is what happens when you add code half asleep 😅

Reply:
Will add a test later for this specific scenario.